charts/vector/templates/NOTES.txt

```diff
@@ -0,0 +1,26 @@
+__   __  __
+\ \ / / / /
+ \ V / / /
+  \_/  \/
+V E C T O R
+
+Vector is starting in your cluster. After a few minutes, you can use Vector's API to view internal metrics by running:
+{{ if (eq .Values.role "Aggregator") }}
+  {{- if not .Values.customConfig }}
+	kubectl -n {{ .Release.Namespace }} exec -it statefulset/{{ include "vector.fullname" . }} -- vector top 
+  {{- else }}
+	kubectl -n {{ .Release.Namespace }} exec -it statefulset/{{ include "vector.fullname" . }} -- vector top --url {{ .Values.customConfig.api.address }}
+  {{- end }}
```

Comment:
Nit, it seems like these are all the same besides the prefix. I might suggest pulling out the calculation of the pod name somewhere where it can be shared (if possible).

Reply:
Indeed - there are a number of places I expect to DRY before (or after) an actual release, checking for role is another. Since it's not necessarily user facing I deferred while I worked on actual functionality (at least until the point it got really annoying to work with 😉 )